addons/hr_timesheet/views/project_task_portal_templates.xml

```diff
@@ -52,12 +52,12 @@
         </xpath>
         <xpath expr="//thead/tr/th[@name='project_portal_milestones']" position="after">
             <t t-if="allow_timesheets">
-                <th t-if="is_uom_day" class="text-end">Days Spent</th>
-                <th t-else="" class="text-end">Hours Spent</th>
+                <th t-if="is_uom_day" class="text-start">Days Spent</th>
+                <th t-else="" class="text-start">Hours Spent</th>
             </t>
         </xpath>
         <xpath expr="//tbody/t/tr/td[@name='project_portal_milestones']" position="after">
-            <td t-if="not project or project.allow_timesheets" class="text-end">
+            <td t-if="not project or project.allow_timesheets" class="text-start">
```

Comment:
Isn't `text-start` the default?

Reply:
Yes, but if we want the title of the column and the column to be align, I found no other solution than doing that :/